app.js

```diff
@@ -1,14 +1,34 @@
 const express = require('express');
-const { getTopics } = require('./controllers/topicsController')
+const { getTopics, getArticlesById } = require('./controllers/controllers')
 console.log(getTopics)
```

Comment:
Remember to clear away console logs when you've finished with them

Reply:
Gone! Goodbye trusty console logs, thanks for helping!